repository/CargoPackageManager-Minimal.package/CGOPackageDependency.class/instance/satisfyConstraints..st

```diff
@@ -0,0 +1,3 @@
+constrains
+satisfyConstraints: aSemanticVersion 
```

Comment:
it is strange to have an argument named aSemanticVersion. I would expect a list of constraints